public/js/lesson-lists.js

```diff
@@ -0,0 +1,50 @@
+$(function() {
+  initializeLocalStorage();
+  let modulePreference = getModulePreference();
+  activateButton(modulePreference);
+  showOrHideLessonContainer(modulePreference, 0);
+
+  $('.module-selector').on('click', function(event) {
+    let previousModulePreference = getModulePreference();
+    deactivateButton(previousModulePreference);
+
+    let moduleSelection = $(event.target).data('module');
+    setModulePreference(moduleSelection);
+    activateButton(moduleSelection);
+    showOrHideLessonContainer(moduleSelection, 100);
+  });
+
+  function initializeLocalStorage() {
+    if (localStorage.getItem('fe-curriculum-lessons') === null) {
+      localStorage.setItem('fe-curriculum-lessons', JSON.stringify({showModule: 0}));
+    }
+  };
+
+  function showOrHideLessonContainer(moduleSelection, ease) {
```

Comment:
I feel a little confused trying to keep track of `modulePreference` and `moduleSelection` here. Is there an issue with converting these functions to arrow functions so that they hang onto their context? In which case we wouldn't have to pass in `modulePreference` as a param, we could just access it directly within the function, since it's defined at the top of the IIFE.

Reply:
I assume we're not using babel, which is likely an issue. Butttt, are we really supporting browsers that are old as shit and don't support ES6 context? Are we using ES6 anywhere else?